.github/actions/prepare-repo-variables/action.yml

```
@@ -0,0 +1,202 @@
+        echo "PACKAGE_NAME_PREVIEW: ${PACKAGE_NAME_PREVIEW}"
+        echo "IMAGE_NAME_PREVIEW: ${IMAGE_NAME_PREVIEW}"
+        
+        PACKAGE_NAME_CACHE="${repo_path}/${repo_path}-cache"
+        CACHE_IMAGE_NAME="ghcr.io/${repo_full_lowercase}/${repo_path}-cache"
+        echo "PACKAGE_NAME_CACHE: ${PACKAGE_NAME_CACHE}"
+        echo "CACHE_IMAGE_NAME: ${CACHE_IMAGE_NAME}"
+        
+        # Set Fly.io app name
+        FLY_APP_NAME="suddenlygiovanni-dev-${PR_NUMBER}"
+        echo "FLY_APP_NAME: ${FLY_APP_NAME}"
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)